book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :book_rentals
+
```

Comment:
- For better readability, the `book_rentals` could be simply called rentals. please, fix this.

Reply:
Done!